Cargo.toml

```diff
@@ -25,11 +25,12 @@ freetype-rs = "0.26"
 pkg-config = "0.3"
 
 [target.'cfg(target_os = "macos")'.dependencies]
-cocoa = "0.24"
+cacao = "0.2.1"
```

Comment:
Could we move without this dependency? Or is there a particular reason for it? What was the issue with e.g. `cocoa`? I'm skeptical since we have both of the in dependency graph in e.g. alacritty, since that's what being used by winit/glutin.

Reply:
cacao provides a safe interface to interact with user defaults, while cocoa requires unsafe. I can switch if it's a blocker, but personally I think cacao is worth it. Up to you.

Reply:
So, I remember now the other reason I used cacao... The cocoa crate only implements setting and reading boolean user defaults, and the type of this value is an int. I agree that cacao should be avoided for now, having looked more closely at the state of each, but I may need to do something special here. https://docs.rs/cocoa/0.24.0/cocoa/foundation/trait.NSUserDefaults.html